server/src/site/xdoc/feature-protocols.xml

```diff
@@ -78,6 +78,20 @@
         Note that MOVE extension (RFC-6851) needs a MailboxManager implementing the move function. For now, this is only done with
         the CassandraMailboxManager
       </p>
+      <p>
+        It supports the METADATA extension (RFC-5464) but
+        <ul>
+          <li>
+            It does not support the server's metadata so far.
+          </li>
+          <li>
+            It does not support permission on metadata cause the ACL extension is not present.
+          </li>
+          <li>
+            It's had the limitation values which is hardcode instead of configurable. Those are the number of annotations on mailbox (10) and the maximum size of an annotation (1024 bytes).
```

Comment:
First sentence is not grammatically correct... 
I would propose "The quota limitations on MailboxAnnotations are hardcoded. They concern the number of annotations per mailbox, as well as the size of an annotation (1024 bytes).